pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -400,9 +404,60 @@ func (r *ReconcileMemberList) Reconcile(request reconcile.Request) (reconcile.Re
 		}
 	}
 
+	// tell Kiali about all the namespaces in the mesh
+	r.reconcileKiali(instance.Namespace, requiredMembers, reqLogger)
```

Comment:
I think you should be using `Status.ConfiguredMembers` here, as those are the namespaces that have been configured for use with the mesh.  It's possible that the required list contains namespaces that don't exist (although, the validator should catch those, but not if the namespace is deleted after creation).

Reply:
Instead of "requiredMembers" getting passed in, this now gets instance.Status.ConfiguredMembers (type changes from map to array - so the code changes to reflect this type change). See: https://github.com/Maistra/istio-operator/pull/87/files#diff-3029a0e7a4af7d79dcfe22782cf7dbbaR408